### PR TITLE
Misc typo fixes

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
lib/ansible/cli/__init__.py
lib/ansible/cli/config.py
lib/ansible/config/manager.py
lib/ansible/modules/cloud/alicloud/ali_instance.py
lib/ansible/modules/cloud/alicloud/ali_instance_facts.py
lib/ansible/modules/cloud/amazon/aws_api_gateway.py
lib/ansible/modules/cloud/amazon/cloudfront_distribution.py
lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group_facts.py
lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py
lib/ansible/modules/cloud/azure/azure_rm_image.py
lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py
lib/ansible/modules/cloud/azure/azure_rm_rediscache.py
lib/ansible/modules/cloud/azure/azure_rm_trafficmanagerendpoint_facts.py
lib/ansible/modules/cloud/azure/azure_rm_webapp_facts.py
lib/ansible/modules/cloud/cloudscale/cloudscale_server.py
lib/ansible/modules/cloud/google/gcp_forwarding_rule.py
lib/ansible/modules/cloud/google/gcp_healthcheck.py
lib/ansible/modules/cloud/google/gcpubsub.py
lib/ansible/modules/cloud/misc/cloud_init_data_facts.py
lib/ansible/plugins/lookup/aws_ssm.py
lib/ansible/plugins/strategy/linear.py
lib/ansible/vars/manager.py
test/units/plugins/test_plugins.py

### CODE REVIEW

File: lib/ansible/cli/adhoc.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def _play_ds(self, pattern, async_val, poll):
         mytask = {'action': {'module': self.options.module_name, 'args': parse_kv(self.options.module_args, check_raw=check_raw)}}
 
         # avoid adding to tasks that don't support it, unless set, then give user an error
-        if self.options.module_name not in ('include_role', 'include_tasks') or any(async_val, poll):
+        if self.options.module_name not in ('include_role', 'include_tasks') or any([async_val, poll]):
```
Comment (Akasurde) [inline lib/ansible/cli/adhoc.py]: @bcoca Could you please review this line ? Thanks.

**Comment by acozine** [comment]
@Akasurde thanks for finding all those typos. If you remove the change to `lib/ansible/cli/adhoc.py`, I can merge the rest of the changes right away.

**Comment by Fred-sun** [comment]
@Akasurde Thanks for your contribution, when you finished changed, I will push for review and merged. Thanks!

**Review by yuwzho — APPROVED** [review]

**Comment by Akasurde** [comment]
@yuwzho @Fred-sun @acozine Thanks for the review.

**Pull request merged**